modules/ossm-cert-manager-integration-istio.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-security.adoc
+
+:_content-type: CONCEPT
+[id="ossm-cert-manager-integration-istio_{context}"]
+== Integrating Service Mesh with cert-manager and istio-csr
```

Comment:
```suggestion
= Integrating Service Mesh with cert-manager and istio-csr
```

Reply:
* This must be an H1. Rely on `leveloffset` in the assembly to place it correctly in the hierarchy.
* If this is a concept, its title should begin with a noun phrase. However, it looks like this is actually the beginning of a three-part procedure. I'd run this by @bburt-rh for additional feedback.